pkgs/servers/baserow/default.nix

```diff
@@ -2,11 +2,20 @@
 , fetchFromGitLab
 , makeWrapper
 , python3
+, antlr4_9
 }:
 
 let
 
-  baserow_premium = with python3.pkgs; ( buildPythonPackage rec {
+  python = python3.override {
+    packageOverrides = self: super: {
+      antlr4-python3-runtime = super.antlr4-python3-runtime.override {
+        antlr4 = antlr4_9;
+      };
+    };
+  };
+
+  baserow_premium = with python.pkgs; ( buildPythonApplication rec {
```

Comment:
Why not add this to the overrides and use `buildPythonPackage`?

Reply:
I'm not sure what baserow_premium is, it's only used in checkInputs. I'm leaving that to @onny.

Reply:
It used `buildPythonPackage` before after all.

Reply:
`baserow_premium` is a dependency of `baserow`. Not sure how to handle this properly, I'm free for any recommendations :)

Reply:
Reverted to `buildPythonPackage` and integrated into the packageOverrides.